its/plugin/src/test/java/com/sonar/python/it/plugin/TestReportTest.java

```diff
@@ -26,6 +26,8 @@
 import com.sonar.orchestrator.Orchestrator;
 import com.sonar.orchestrator.build.BuildResult;
 import com.sonar.orchestrator.build.SonarRunner;
+
+import org.junit.Before;
```

Comment:
@vilchik-elena You are right, we need to have identical settings. I am using the preferences from https://github.com/SonarSource/sonar-developer-toolset. Looks like the basic Eclipse settings are not modified by the SonarSource prefs. We should check that together, because it causes noisy changes.